cmds/grpc-backend/main.go

```diff
@@ -107,18 +107,28 @@ func createRIDServer(ctx context.Context, locality string, logger *zap.Logger) (
 		return nil, stacktrace.Propagate(err, "Failed to connect to remote ID database; verify your database configuration is current with https://github.com/interuss/dss/tree/master/build#upgrading-database-schemas")
 	}
 
+	ridStore, err := ridc.NewStore(ctx, ridCrdb, logger)
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "Failed to create remote ID store")
+	}
+
+	repo, err := ridStore.Interact(ctx)
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "Unable to interact with store")
+	}
+	gc := ridc.NewGarbageCollector(repo, locality)
+
 	// schedule period tasks for RID Server
 	ridCron := cron.New()
 	// schedule pinging every minute for the underlying storage for RID Server
 	if _, err := ridCron.AddFunc("@every 1m", func() { pingDB(ctx, ridCrdb, ridc.DatabaseName) }); err != nil {
 		return nil, stacktrace.Propagate(err, "Failed to schedule periodic ping to %s", ridc.DatabaseName)
 	}
-	ridCron.Start()
 
-	ridStore, err := ridc.NewStore(ctx, ridCrdb, logger)
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "Failed to create remote ID store")
+	if _, err = ridCron.AddJob("@every 30m", GarbageCollectorJob{"delete rid expired records", *gc, ctx}); err != nil {
```

Comment:
Ideally we would want to make the 30m configurable we can do this in the next PR